content/02-contributions-to-existing-projects.md

```diff
@@ -1,4 +1,4 @@
-# How to contribute to OSS projects
+# How to contribute to open source projects
```

Comment:
OSS = open source software
Maybe capitalize the O S and S as it is on the first occurrence in the README